imap_processing/spice/repoint.py

```diff
@@ -193,3 +193,21 @@ def interpolate_repoint_data(
     out_df["repoint_in_progress"] = query_met_times < out_df["repoint_end_met"].values
 
     return out_df
+
+
+def get_pointing_start_time(query_met_time: float) -> tuple[float, float]:
```

Comment:
Remove "query" from the variable arguments to the function, you are just taking in a met_time.
```suggestion
def get_pointing_start_time(met_time: float) -> tuple[float, float]:
```